pkg/dockerregistry/server/auth.go

```diff
@@ -90,7 +91,7 @@ var (
 )
 
 func newAccessController(options map[string]interface{}) (registryauth.AccessController, error) {
-	log.Info("Using Origin Auth handler")
+	log.Info("using Origin Auth handler")
```

Comment:
Well, most of the upstream log messages are lower-cased. In v2.4.1 it's 33 messages starting with upper-cased letter vs 81 starting with lower-cased (all log levels).

But ok, keeping the messages same won't do any harm. I'll lower-case the new messages though.